src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -606,6 +606,7 @@ public C P
                 });
 
             compilation.VerifyDiagnostics();
+            compilation.GetEmitDiagnostics();
```

Comment:
Consider making this method a regular `Fact` now that the UsedAssemblies issue is fixed.